long-island-iced-tea/bangazon
# Mc computer makemodel

Adds make and model to database,
updates model, and update and post methods

# To Test: 
- Run sql script
- make a get request and see new data
